R/init.R

```diff
@@ -58,23 +58,22 @@ copy_assets <- function(pkg = ".") {
 
   # pkgdown assets
   if (!identical(template$default_assets, FALSE)) {
-    copy_asset_dir(pkg, path_pkgdown(paste0("BS", pkg$bs_version), "assets"))
-  }
-
-  # manually specified directory: I don't think this is documented
-  # and no longer seems important, so I suspect it could be removed
-  if (!is.null(template$assets)) {
-    copy_asset_dir(pkg, template$assets)
+    copy_asset_dir(
+      pkg, 
+      path_pkgdown(paste0("BS", pkg$bs_version, "/", "assets")), 
+      src_root = path_pkgdown(),
+      src_label = "<pkgdown>/"
+    )
   }
 
   # package assets
   if (!is.null(template$package)) {
-    assets <- path_package_pkgdown(
-      "assets",
-      package = template$package,
-      bs_version = pkg$bs_version
+    copy_asset_dir(
+      pkg, 
+      path_package_pkgdown("assets", template$package, pkg$bs_version),
+      src_root = system_file(package = template_package),
```

Comment:
Since this was merged, we get:

```
Error: 
! in callr subprocess.
Caused by error in `copy_assets(pkg)`:
! object 'template_package' not found
```

Any chance this was meant to be?

```r
src_root = system_file(package = template$package),
```

I can open a new issue if you prefer.

Reply:
Oops yes. Let me push a fix real quick